### ISSUE TITLE
unable to delete "npp.7.7.1.Installer.x64.exe" file after auto update

### ISSUE THREAD

**Issue by kedi1992** [notepad-plus-plus/notepad-plus-plus#5963]
### Description of the Issue
I have upgraded notepad successfully. 
but I observed, "npp.7.7.1.Installer.x64.exe" auto-downloaded under temp folder and it's not getting deleted after upgrade

"npp.7.7.1.Installer.x64.exe" taking more than 3MB space 
 

### Steps to Reproduce the Issue

1. Open Nodepad ++
2. Upgrade Nodepad++
3. Check tmp folder after upgrade 

### Expected Behavior
after upgrade, "npp.7.7.1.Installer.x64.exe" notepad++ installer file should delete

### Actual Behavior
"npp.7.7.1.Installer.x64.exe" is not getting deleted 

OS: Windows 64bit
Temp folder path : C:\Users\pravin.k\AppData\Local\Temp\npp.7.7.1.Installer.x64.exe

**Comment by wesinator** [comment]
yes, it doesn't remove the update installer from %TEMP% after installing.

see also https://github.com/notepad-plus-plus/notepad-plus-plus/pull/2309
